docarray/array/storage/qdrant/find.py

```diff
@@ -1,23 +1,16 @@
 from abc import abstractmethod
-from typing import (
-    TYPE_CHECKING,
-    TypeVar,
-    Sequence,
-    List,
-    Union,
-    Optional,
-    Dict,
-)
-from qdrant_client.http.models.models import Distance
+from typing import TYPE_CHECKING, Dict, List, Optional, Sequence, TypeVar, Union
 
 from docarray import Document, DocumentArray
 from docarray.math import ndarray
 from docarray.score import NamedScore
+from qdrant_client.http import models as rest
```

Comment:
did you check if this is supported in lowest qdrant version we have ?
https://github.com/jina-ai/docarray/blob/main/setup.py#L68
If not, we might have to increase the version or import locally

Reply:
okay let me check it